src/components/ComponentPages/DirectSupportedCamps/index.tsx

```diff
@@ -13,11 +14,25 @@ const DirectSupportedCamps = () => {
   const RemoveCardSupportedCamps = () => {
     setIsSupportedCampsModalVisible(true);
   };
+
+  const fetchDirectSupportedCampsList = async () => {
+    let response = await getDirectSupportedCampsList();
+    if (response && response.status_code === 200) {
```

Comment:
I think the `response.status_code === 200` will work and we can remove the `response` inside `if` condition. No?

Reply:
response could be null or empty or undefined in case of connection error or any other reason. For that reason I have applied it inside the if condition